js_modules/dagster-ui/packages/ui-core/src/asset-graph/sidebar/Sidebar.tsx

```diff
@@ -161,17 +162,32 @@ export const AssetGraphExplorerSidebar = React.memo(
       });
 
       if (groupsCount === 1) {
-        return folderNodes
-          .filter((node) => node.level === 3)
-          .map((node) => ({
-            ...node,
-            level: 1,
-          }));
+        return {
+          folderNodes: folderNodes
+            .filter((node) => node.level === 3)
+            .map((node) => ({
+              ...node,
+              level: 1,
+            })),
+          groupsCount,
+        };
       }
 
-      return folderNodes;
+      return {folderNodes, groupsCount};
     }, [graphData.nodes, openNodes]);
 
+    const {nav} = React.useContext(LayoutContext);
+
+    React.useEffect(() => {
+      if (groupsCount > 1) {
+        // Close the left sidebar if there's more than 1 group (if viewing more than 1 group then this is the global asset graph)
+        // to avoid duplication between the global left sidebar and the graph's left sidebar.
+        nav.close();
+      }
+      // Exclude groupsCount so that we don't close the left nav due to filtering changing
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [nav.close]);
```

Comment:
Would it be safe to say that you'd only want this on initial load, since changes to `groupsCount` won't affect anything? If so, you could just use the empty dependency array and eliminate the lint bypass.